package.json

```diff
@@ -52,6 +52,7 @@
     "react-router": "^2.4.1",
     "react-router-bootstrap": "^0.23.1",
     "react-router-scroll": "^0.2.0",
+    "react-tooltip": "^3.2.6",
```

Comment:
Jeg synes ikke vi skal legge inn et nytt bibliotek, når vi allerede bruker react-bootstrap, og den har all funksjonaliteten vi trenger.

For å bruke dette må du først slå det på (skulle vel egentlig vært slått på fra før). Sørg for at følgende linjer er satt til `true` i .bootstraprc (i rot-folderen):
```
  modals: true
  tooltip: true
  popovers: true
  carousel: true
```

(vi trenger bare tooltip her, men vi kan like godt sette alle til true).